src/WebSocketsServiceProvider.php

```diff
@@ -26,23 +27,30 @@ public function boot()
             __DIR__.'/../config/websockets.php' => base_path('config/websockets.php'),
         ], 'config');
 
-        if (! Schema::hasTable('websockets_statistics_entries')) {
-            $this->publishes([
-                __DIR__.'/../database/migrations/create_websockets_statistics_entries_table.php.stub' => database_path('migrations/'.date('Y_m_d_His', time()).'_create_websockets_statistics_entries_table.php'),
-            ], 'migrations');
-        }
+        try {
+            if (! Schema::hasTable('websockets_statistics_entries')) {
+                $this->publishes([
+                    __DIR__.'/../database/migrations/create_websockets_statistics_entries_table.php.stub' => database_path('migrations/'.date('Y_m_d_His', time()).'_create_websockets_statistics_entries_table.php'),
```

Comment:
Instead of generating the file date and time suffix, it might be a good idea to compute one with `000000` and change the extension to `.php` from `.php.stub`

Reply:
Updated to `000000`, however change of extension not required as `.php.stub` is copied into `.php` file

Reply:
It's not what I meant, but I'll fix it for you.

Reply:
Thanks, understood it better 👍